examples/app/polymer/polymer.controller.js

```diff
@@ -0,0 +1,68 @@
+angular.module('makeithappen').controller('PolymerController', function() {
+    var vm = this;
+    vm.schema = {
```

Comment:
I don't know whether the schemas are identical, but we probably should unify all/most of them. I'd prefer to track that via a separate issue however.